.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,25 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+permissions:
```

Comment:
Google recommends `permissions: read-all`.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#defining-access-for-the-github_token-permissions

Reply:
Updated

Reply:
I don't see it. Do you need to `git push`?